dspace/config/modules/coar-notify-ldn.cfg

```diff
@@ -0,0 +1,46 @@
+#---------------------------------------------------------------#
+#---------------COAR NOTIFY LDN CONFIGURATION-------------------#
+#---------------------------------------------------------------#
+#     Configuration properties used  by Coar Notify and ldn     #
+#---------------------------------------------------------------#
+
+#Boolean to determine if Coar Notify is enabled globally for entire site.
+#default => false
+coar-notify.enabled = false
```

Comment:
Let me better describe the authorization system here. The LDN management system produces a new source for QA Events: this group is added to the supported sources at org.dspace.qaevent.service.impl.QAEventServiceImpl.getSupportedSources() [here](https://github.com/4Science/DSpace/blob/1f438d1665d370278f682b1960414a9e3e362b87/dspace-api/src/main/java/org/dspace/qaevent/service/impl/QAEventServiceImpl.java#L690-L693)
  
_coar-notify QA events_ are always visible to the admin;
_coar-notify QA events_ are visible to non-admin users only if they are submitters of the referenced Item of the event;

_ldn.enabled_ property activates the LDN Message Inbox: ldn messages are received on the configured url (see property ldn.notify.inbox at [here](https://github.com/4Science/DSpace/blob/1f438d1665d370278f682b1960414a9e3e362b87/dspace/config/modules/coar-notify-ldn.cfg#L19) 
**true** => message is received and managed
**false** => message is refuced and the server respond with an HTTP 404

_coar-notify.enabled_ property is used in the authorization feature at org.dspace.app.rest.authorization.impl.CoarNotifyEnabled (and is used to show/hide the COAR logo on the frontend footer). This authorization feature is used to show/hide the "LDN Service" option on the sidebar menu.

We decoupled the configuration in two different properties because we want for the admin to configure LDN Service while its inbox is disabled.